lib/test-service-provider.js

```diff
@@ -64,15 +72,17 @@
 				return service;
 			}
 
-			// Return a real registered service, if it exists (sometimes you want the real deal, i.e. utils)
-			preRegisteredService = application.getServiceForTest(serviceName, this);
-			if (preRegisteredService) {
-				return preRegisteredService;
+			if (this.allowedServices.hasOwnProperty(serviceName)) {
```

Comment:
This was intentional, though maybe too heavy handed. I figure that most people aside from Box shouldn't have too much trouble adding a whitelist to their tests. Debugging issues will also be pretty easy since the error contains the service name.

Making this optional whitelist allows for the some tests to slip through with the same problems